### PR TITLE
Feature/rebranding

### PR DESCRIPTION
**Pull request by btomala**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -2,13 +2,12 @@
...
  ///
 
-import ScalaxbKeys._
 import scalariform.formatter.preferences._
 import sbtassembly.{MergeStrategy, PathList}
 
 name := "scoozie"
 
-organization := "com.klout"
+organization := "com.virtuslab"
```
Comment (explicite) [inline build.sbt]: org.virtuslab

**Review by explicite — CHANGES_REQUESTED** [review]
replace `com.virtuslab` to `org.virtuslab`

**Review by explicite — APPROVED** [review]

File: src/test/scala/XmlGenerationSpec.scala [hunk 2]
```diff
@@ -2,14 +2,16 @@
  * Copyright (C) 2013 Klout Inc. <http://www.klout.com>
```
Comment (explicite) [inline src/test/scala/XmlGenerationSpec.scala:2]: maye we can change this too? (and update year)

Comment (btomala) [reply src/test/scala/XmlGenerationSpec.scala:2]: What should I put there?

Comment (explicite) [reply src/test/scala/XmlGenerationSpec.scala:2]: maybe `Copyright (C) 2018 Virtus Lab sp. z o.o. (Ltd.) <https://virtuslab.com>`

Comment (btomala) [reply src/test/scala/XmlGenerationSpec.scala:2]: `virtuslab.org`?

**Pull request closed without merging**